frontend/appflowy_flutter/lib/shared/icon_emoji_picker/recent_icons.dart

```diff
@@ -0,0 +1,89 @@
+import 'dart:convert';
+
+import 'package:appflowy/shared/icon_emoji_picker/icon.dart';
+import 'package:appflowy_backend/log.dart';
+import 'package:flutter/foundation.dart';
+
+import '../../core/config/kv.dart';
+import '../../core/config/kv_keys.dart';
+import '../../startup/startup.dart';
+import 'flowy_icon_emoji_picker.dart';
+
+class RecentIcons {
+  static final Map<String, List<String>> _dataMap = {};
+  static bool _loaded = false;
+  static const maxLength = 20;
+  @visibleForTesting
+  static bool enable = true;
+
+  static Future _save() async {
```

Comment:
Please declare return types always:
```suggestion
  static Future<void> _save() async {
```
Same for all methods in this class.